helm/kubernetes-cluster-prep/test-helm

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Usage:
+#     ./test-helm [<Helm test release>]
+#
+# Example:
+#     ./test-helm my-test-release
+
+print_usage() {
+  echo "Usage:"
+  echo "    This script will test the Helm release"
+  echo ""
+  echo "Syntax:"
+  echo "    $0 [Options]"
+  echo "    Options:"
+  echo "    -h           Show help"
+  echo "    -i           Install the Helm release"
+  echo "    -d           Unitall the Helm release"
+  echo "    -r           Specify the Helm release"
+  echo "    -None at this time"
+}
+
+helm_release="helm-test-release"
+install=false
+uninstall=false
+conjur_account="${CONJUR_ACCOUNT:-myConjurAccount}"
+conjur_appliance_url="${CONJUR_APPLIANCE_URL:-https://conjur-oss.conjur-oss.svc.cluster.local}"
+conjur_certificate_file_path="${CONJUR_CERT_FILE_PATH:-files/conjur-cert.pem}"
+authenticator_id="${AUTHENTICATOR_ID:-my-authenticator-id}"
+
+function main() {
+  # Process command line options
+  local OPTIND
+  while getopts ':d:h:ir:u' flag; do
+    case "${flag}" in
+      #a) conjur_account=${OPTARG} ;;
+      h) print_usage; exit 0 ;;
+      i) install=true ;;
+      r) helm_release=${OPTARG} ;;
+      d) uninstall=true ;;
+      u) uninstall=true ;;
```

Comment:
We have both `u` and `d` options, maybe `u` would be clearer?

Reply:
Agreed